kernel/src/main/java/org/kframework/parser/inner/ParseInModule.java

```diff
@@ -186,6 +186,10 @@ public Scanner getScanner() {
         return scanner;
     }
 
+    public void setScanner(Scanner scanner) {
```

Comment:
This seems suspicious to me.

Reply:
I needed this because I'm now precomputing the scanners before I have all the parsers.
When I get to create the parser, I just lookup for the appropriate scanner and do parsing.